docs/source/operations/projections/s2.rst

```diff
@@ -0,0 +1,109 @@
+.. _s2:
```

Comment:
A reference to that file should be added in docs/source/operations/projections/index.rst